index.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Lacquer&family=Poppins&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+}
+
+body {
+  background-image: url('./img/background_image.png');
+}
+
+a,
+.second-section ul {
+  text-decoration: none;
+  list-style: none;
+}
+
+.header-bar {
+  display: flex;
+  padding: 20px;
+  font-size: 15px;
+}
+
+.header-bar header {
+  flex: 3;
+  font-size: 18px;
+  color: #6070ff;
+  font-weight: bold;
+}
+
+.main-section h1 {
+  font-weight: 700;
+  color: #172b4d;
+  font-size: 40px;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  margin: 5% 25%;
+}
+
+.main-section > * {
+  padding: 20px;
+}
+
+.main-section p {
+  color: #344563;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.second-section ul {
+  display: flex;
+  flex-direction: row;
+  margin: 10px 0;
+}
+
+.second-section ul li {
+  padding: 10px;
+  margin-left: -10px;
+}
+
+.second-section ul li a img:hover {
+  background-color: rgb(182, 182, 233);
+}
+
+.second-section h3 {
+  color: #7f8cff;
+  font-size: 16px;
+}
+
+.header-bar div {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding: 2px;
+  border-radius: 3px;
+  background: white;
+}
+
+.three-bar-menu span {
+  display: block;
+  width: 25px;
+  height: 2px;
+  border-radius: 3px;
+  background-color: #6070ff;
+}
+
+@media all and (min-width: 375px) {
+  body {
+    font-size: 16px;
+  }
+}
+
+@media all and (max-width: 768px) {
+  /* .header-bar ul {
+    display: none;
+  } */
+
+  /* .header-bar header {
+    text-align: center;
+  } */
```

Comment:
- [ ] [OPTIONAL] It is not a good practice to leave unused pieces of code as comments, kindly remove them to keep the codebase clean and professional. 🤓️